inc/dbutils.class.php

```diff
@@ -909,57 +909,83 @@ public function getTreeLeafValueName($table, $ID, $withcomment = false, $transla
       $name    = "";
       $comment = "";
 
-      $SELECTNAME    = "`$table`.`name`, '' AS transname";
-      $SELECTCOMMENT = "`$table`.`comment`, '' AS transcomment";
-      $JOIN          = '';
+      $SELECTNAME    = new \QueryExpression("'' AS ". $DB->quoteName('transname'));
+      $SELECTCOMMENT = new \QueryExpression("'' AS " . $DB->quoteName('transcomment'));
+      $JOIN          = [];
+      $JOINS         = [];
       if ($translate) {
          if (Session::haveTranslations($this->getItemTypeForTable($table), 'name')) {
-            $SELECTNAME  = "`$table`.`name`, `namet`.`value` AS transname";
-            $JOIN       .= " LEFT JOIN `glpi_dropdowntranslations` AS namet
-                              ON (`namet`.`itemtype` = '".$this->getItemTypeForTable($table)."'
-                                 AND `namet`.`items_id` = `$table`.`id`
-                                 AND `namet`.`language` = '".$_SESSION['glpilanguage']."'
-                                 AND `namet`.`field` = 'name')";
+            $SELECTNAME = new \QueryExpression("namet.value AS ". $DB->quoteName('transname'));
```

Comment:
I think the usage of `QueryExpression` is useless here.
```suggestion
            $SELECTNAME = 'namet.value AS transname';
```